src/components/Bookinput.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Bookinput() {
+  return (
+    <form>
+      <input type="text" placeholder="book title" />
+      <button type="button">submit</button>
+    </form>
+  );
+}
+
+export default Bookinput;
```

Comment:
- [ ] It should take the book data as a prop and renders the title and the author. please implement it